core/src/main/java/fj/Monoid.java

```diff
@@ -1109,6 +1102,28 @@ public Unit append(Unit a1, Unit a2) {
     });
   }
 
+  /**
+   * A intersection monoid for sets.
+   *
+   * @param bounded A bound for all possible elements
+   * @param enumerator An enumerator for all possible elements
+   * @param o An order for set elements.
+   * @return A monoid for sets whose elements have the given order.
+   */
+  public static <A> Monoid<Set<A>> setIntersectionMonoid(final Bounded<A> bounded, final Enumerator<A> enumerator, final Ord<A> o) {
```

Comment:
here you can remove the `final Ord<A>` parameter since it is available through `enumerator.order()`.